apps/src/templates/accessible-dialogue.module.scss

```diff
@@ -6,19 +8,36 @@
   left: 0;
   background-color: #000;
   opacity: 0.6;
-  z-index: 1000;
+  z-index: 1250;
```

Comment:
Had to alter these z indexes to get it to grey out the instructions that are highlighted behind this popup